wiki/en/en-Server-Admin-Manual.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: wiki
+title: "Server Adminstrator's Manual"
+lang: "en"
+permalink: "/wiki/Server-Admin-Manual"
+---
+
+# Jamulus Server Adminstrator's Manual
```

Comment:
Administrator's instead of Adminstrator's

Reply:
I think it might also need to be "Administrator" not "Administrator's", since there will also be "User Manual".